### DISCUSSION TITLE
Маленькие недочёты

### DISCUSSION THREAD

**Discussion by lichnost3d** [mikopbx/Core#227] [General ( Обо всём )]
Только начал знакомиться с MikoPBX, продукт очень нравится своей простотой, но есть пару вещей которых прям не хватает:
1) В справочнике "Сотрудники"  полей
-"Должность"
-"Отдел"
-"Дата создания"
Без них, неудобно ориентироваться в списке больше чем 20 человек
2) В справочнике "Сотрудники" не хватает пагинации

**Comment by boffart** [comment]
"**Отдел**" - это поле ближе всего с модулю "[Группы пользователей](https://wiki.mikopbx.com/module-users-groups)"
**Должность** - Мб. тут проще реализовать что то вроде "**Комментарий**", как это сделано в списке провайдеров. Но под вопросом, это все же не учетная система, а АТС. 

"Пагинация", то есть страницы? Это можно реализовать.

**Comment by lichnost3d** [comment]
Это всё сугубо ради удобства, зачастую непонятно, что это за сотрудник, из какого он отдела, когда он был заведен в справочнике?
Если сложно добавить поле "Должность" то хотя бы "Комментарий" добавить,уже будет гораздо проще.
А дата создания (в идеале и дата последнего изменения) и пагинация на ваш взгляд, это уже избыточные вещи?

**Comment by boffart** [comment]
Добавить страницы - это удобная вещь, согласен, а вот даты, тут пока не вижу смысла.

**Comment by jorikfon** [comment]
А какая цель отображения постранично? Щас все от этого избавляются, в соцсетях у вас бесконечная лента, список пользователей достаточно легкая информация даже если их сотни. Фильтр по имени, номеру работает сейчас. 

Комментарий подумаем, дополнительные поля, тоже, может модули научим такое делать.
